Makefile

```diff
@@ -1044,6 +1044,18 @@ grpc:
 grpc/host: protos/all
 	@build.assets/genproto.sh
 
+# protos-up-to-date checks if the generated GRPC stubs are up to date.
+# This target runs in the buildbox container.
+.PHONY: protos-up-to-date
+protos-up-to-date:
+	$(MAKE) -C build.assets protos-up-to-date
+
+# protos-up-to-date/host checks if the generated GRPC stubs are up to date.
+# Unlike protos-up-to-date, this target runs locally.
+.PHONY: protos-up-to-date/host
+protos-up-to-date/host:
```

Comment:
Could this be as simple as depending on grpc/host, with a git status check inside?

Reply:
+1

Reply:
This change sort of re-imagines what this script does, so I think what I'm going to do is call this script `is-repo-dirty.sh` and have the `Makefile` utilize it to determine if the repo is dirty after running `grpc/host`.

Reply:
Alright, that is in place now.